### PR TITLE
Add information about changing the content source of a host

### PR DESCRIPTION
**Pull request by AkshayGadhaveRH**
Added a new module proc_changing_the_content_source_of_a_host.adoc to add information about changing the content source of a host in the Managing Hosts doc.

Cherry-pick into:

* [ ] Foreman 3.1
* For Foreman 3.0 or older, file a separate PR request

@melcorr

### CODE REVIEW

File: guides/common/modules/proc_changing_the_content_source_of_a_host.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[id="Changing_the_Content_Source_of_a_Host{context}"]
+= Changing the Content Source of a Host
+
+A Content Source is a {SmartProxy} that a host consumes content from. Use this procedure to change the Content Source of a Host
```
Comment (melcorr) [inline guides/common/modules/proc_changing_the_content_source_of_a_host.adoc:4]:
```suggestion
A content source is a {SmartProxy} that a host consumes content from. Use this procedure to change the content source of a host
```

There are a few terms that are unique terms related to the project and they require capitalization. However, all other terms are just common nouns. A content source is just a source of content, not a real product term. It's a very common mistake to capitalize all terms in a project, but not necessary.

File: guides/common/modules/proc_changing_the_content_source_of_a_host.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All hosts*.
+. Select the check box of the host or hosts you want to change.
```
Comment (melcorr) [inline guides/common/modules/proc_changing_the_content_source_of_a_host.adoc:8]:
```suggestion
. Select the check box of one or more hosts whose content source you want to change. 
```

File: guides/common/modules/proc_changing_the_content_source_of_a_host.adoc [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All hosts*.
+. Select the check box of the host or hosts you want to change. 
+. From the *Select Action* list, select *Change Content Source*.
+. From the *Environment* list, select the new environment for your host.
+. From the *Content View* list, select the new Content View for your host.
+. From the *Content Source* list, select the new {SmartProxy} for your host.
+. Click *Change content source*. 
+
+You can either complete the content source change using remote execution or manually. To update configuration on host using remote execution, click on *Run job invocation*. To update the content source manually, enter the autogenerated commands in *Change content source* on the host.
```
Comment (melcorr) [inline guides/common/modules/proc_changing_the_content_source_of_a_host.adoc:15]: Please provide an xref to https://docs.theforeman.org/3.1/Managing_Hosts/index-foreman-el.html#executing-a-remote-job_managing-hosts

**Review by melcorr — CHANGES_REQUESTED** [review]
Thank you @AkshayGadhaveRH - almost there. 

@stejskalleos - can you take a look also and suggest any necessary changes?

**Comment by stejskalleos** [comment]
If https://github.com/theforeman/foreman-documentation/pull/1067#discussion_r807735347 will be fixed then it LGTM

File: guides/common/modules/proc_changing-the-content-source-of-a-host.adoc [hunk 4]
```diff
@@ -0,0 +1,16 @@
+[id="Changing_the_Content_Source_of_a_Host{context}"]
+= Changing the Content Source of a Host
+
+A content source is a {SmartProxy} that a host consumes content from. Use this procedure to change the content source of a Host
```
Comment (melcorr) [inline guides/common/modules/proc_changing-the-content-source-of-a-host.adoc:4]:
```suggestion
A content source is a {SmartProxy} that a host consumes content from. 
Use this procedure to change the content source of a Host
```
We follow a strict one sentence per line rule.

File: guides/common/modules/proc_changing-the-content-source-of-a-host.adoc [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All hosts*.
+. Select the check box of one or more hosts whose content source you want to change. 
+. From the *Select Action* list, select *Change Content Source*.
+. From the *Environment* list, select the new environment for your host.
+. From the *Content View* list, select the new Content View for your host.
+. From the *Content Source* list, select the new {SmartProxy} for your host.
+. Click *Change content source*. 
+
+You can either complete the content source change using remote execution or manually. To update configuration on host using remote execution, click on *Run job invocation*. For more information about running remote execution jobs, see {ManagingHostsDocURL}configuring-and-setting-up-remote-jobs_managing-hosts[Configuring and Setting up Remote Jobs]. To update the content source manually, enter the autogenerated commands in *Change content source* on the host.
```
Comment (melcorr) [inline guides/common/modules/proc_changing-the-content-source-of-a-host.adoc:15]:
```suggestion
You can either complete the content source change using remote execution or manually. 
To update configuration on host using remote execution, click on *Run job invocation*. 
For more information about running remote execution jobs, see {ManagingHostsDocURL}configuring-and-setting-up-remote-jobs_managing-hosts[Configuring and Setting up Remote Jobs]. 
To update the content source manually, enter the autogenerated commands in *Change content source* on the host.
```

Comment (melcorr) [reply guides/common/modules/proc_changing-the-content-source-of-a-host.adoc:15]: Same here.

**Review by melcorr — CHANGES_REQUESTED** [review]
Well done on fixing that merge commit error! 
We follow a one-sentence-per-line rule. 
Please see my suggestions. 
Otherwise we are looking good!

File: guides/common/modules/proc_changing-the-content-source-of-a-host.adoc (hunk 5 above)
Comment (melcorr) [reply guides/common/modules/proc_changing-the-content-source-of-a-host.adoc:15]: One edit: s/Click on/Click - as per IBM Style :o)

File: guides/common/modules/proc_changing-the-content-source-of-a-host.adoc [hunk 6]
```diff
@@ -0,0 +1,20 @@
+[id="Changing_the_Content_Source_of_a_Host{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_changing-the-content-source-of-a-host.adoc:1]:
```suggestion
[id="Changing_the_Content_Source_of_a_Host_{context}"]
```

File: guides/common/modules/proc_changing-the-content-source-of-a-host.adoc [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+= Changing the Content Source of a Host
+
+A content source is a {SmartProxy} that a host consumes content from. 
+Use this procedure to change the content source of a Host
```
Comment (maximiliankolb) [inline guides/common/modules/proc_changing-the-content-source-of-a-host.adoc:5]:
```suggestion
Use this procedure to change the content source of a host.
```

File: guides/common/modules/proc_changing-the-content-source-of-a-host.adoc [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All hosts*.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_changing-the-content-source-of-a-host.adoc:8]:
```suggestion
. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
```

File: guides/common/modules/proc_changing-the-content-source-of-a-host.adoc [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+. From the *Environment* list, select the new environment for your host.
+. From the *Content View* list, select the new Content View for your host.
+. From the *Content Source* list, select the new {SmartProxy} for your host.
+. Click *Change content source*. 
+
+You can either complete the content source change using remote execution or manually. 
+To update configuration on host using remote execution, click *Run job invocation*. 
+For more information about running remote execution jobs, see {ManagingHostsDocURL}configuring-and-setting-up-remote-jobs_managing-hosts[Configuring and Setting up Remote Jobs]. 
+To update the content source manually, enter the autogenerated commands in *Change content source* on the host.
+
```
Comment (maximiliankolb) [inline guides/common/modules/proc_changing-the-content-source-of-a-host.adoc:20]:
```suggestion
```

**Review by maximiliankolb — CHANGES_REQUESTED** [review]

**Review by melcorr — APPROVED** [review]
LGTM, thank you!

**Comment by melcorr** [comment]
Cherry-pick:
 0247d92b..abae7174  3.2 -> 3.2

**Pull request merged**